enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -126,7 +126,7 @@ const (
 	minScratchDiskSizeBytes = 64e6
 
 	// Chunk size to use when creating COW images from files.
-	cowChunkSizeBytes = 512 * 1024
+	cowChunkSizeInPages = 1000
```

Comment:
The optimal chunk size might be different from the optimal UFFD copy size, since we'll be copying data to/from cache at chunk granularity. Can we have 2 separate knobs for these? Maybe cowChunkSizeBytes / uffdCopySizeInPages

Reply:
(Also, if we have 2 separate knobs for chunkSize vs uffdCopySize, it might change how the tests perform; may be worth re-running just to double check that 1000 pages is still the optimal number)

Reply:
I'm not sure if this is worth it for now. Right now, the logic is simple because we always copy entire chunks. If we make the copy size != chunk size, we will also need to track which addresses we've already copied (you can't copy the same address twice)

Increasing the copy size changes the speed by ms. Copying 500 pages takes ~1ms, copying 1000 pages takes 2ms. If fetching remote blocks becomes the bottleneck, it seems like we might as well just copy the entire chunk to avoid having to make more round trips

Reply:
I think Tyler recently found that 10MB was the optimal-ish size for transferring files between the executor and the app - 4MB isn't too far off so keeping this for now sgtm

Reply:
Yeah these were the (pretty rough) numbers:

```
	- can upload a 1GB file, compressed, serially, in 16seconds []
	- can upload the same 1GB file but chunked into 10 100M parts, in 1.75 seconds [500MB/sec]
	- can upload the same 1GB file but chunked into 100 10M parts, in 0m2.489s
	- can upload the same 1GB file but chunked into 1000 1M parts, in 11.618s
```